dbaas-lib/engines/k8s-psmdb/types/v110/psmdb_types.go

```diff
@@ -85,6 +85,17 @@ func (cr *PerconaServerMongoDB) Upgrade(imgs map[string]string) {
 	}
 }
 
+func (cr *PerconaServerMongoDB) GetStatus() string {
+	return string(cr.Status.Status)
+}
+func (cr *PerconaServerMongoDB) GetReplestsNames() []string {
```

Comment:
I don't see any usage of these methods anywhere in the code.

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/blob/DBAASCLI-66/dbaas-lib/engines/k8s-psmdb/controller.go#L141
https://github.com/Percona-Lab/percona-dbaas-cli/blob/DBAASCLI-66/dbaas-lib/engines/k8s-psmdb/controller.go#L77